chains/evm/cli/erc20/deposit.go

```diff
@@ -64,6 +64,7 @@ func BindDepositFlags(cmd *cobra.Command) {
 	cmd.Flags().Uint8Var(&DomainID, "domain", 0, "Destination domain ID")
 	cmd.Flags().StringVar(&ResourceID, "resource", "", "Resource ID for transfer")
 	cmd.Flags().Uint64Var(&Decimals, "decimals", 0, "ERC20 token decimals")
+	cmd.Flags().StringVar(&Priority, "priority", "medium", "Transaction priority speed (default: medium)")
```

Comment:
Should we have slow or medium for default? Though, it currently doesn't affect anything it is more of a future question.

Reply:
I thought about the same thing, I went with the "medium" as default, because MetaMask has the same approach and I think it's a golden mean.
I guess if someone has some pros/cons about that he will comment here.